pyuvdata/uvdata/tests/test_mwa_corr_fits.py

```diff
@@ -549,3 +549,31 @@ def test_flag_init_errors(flag_file_init, err_type, read_kwargs, err_msg):
     with pytest.raises(err_type) as cm:
         uv.read(flag_file_init, **read_kwargs)
     assert str(cm.value).startswith(err_msg)
+
+
+def test_read_metadata_only(tmp_path):
+    """
+    MWA correlator fits to uvfits loopback test.
+
+    Read in MWA correlator files, write out as uvfits, read back in and check
+    for object equality.
```

Comment:
This docstring doesn't seem to match the logic below.

Reply:
ah yes thank you, copy/pasta errors.